front/src/common/BootstrapSNCF/SelectSNCF.tsx

```diff
@@ -49,9 +51,10 @@ function SelectSNCF<T extends string | SelectOptionObject>({
         }}
         className={cx(className, sm && 'sm')}
         value={isString(value) ? value : value?.id}
+        disabled={disabled}
       >
-        {options.map((option) => (
-          <option key={getOptionValue(option)} value={getOptionValue(option)}>
+        {options.map((option, index) => (
+          <option key={`${index}-${getOptionValue(option)}`} value={getOptionValue(option)}>
```

Comment:
Why did you add that ? Actually, we should not have any duplicates in the options